tracer/build/_build/Build.Steps.cs

```diff
@@ -1204,6 +1205,193 @@ projectPath switch
             }
         });
 
+    Target CopyLogsToBuildData => _ => _
+       .Unlisted()
+       .Description("Copies logs and coredumps to build_data directory")
+       .Executes(() => MoveLogsToBuildData());
+
+    Target CheckBuildLogsForErrors => _ => _
+       .Unlisted()
+       .Description("Reads the logs from build_data and checks for error lines")
+       .Executes(() =>
+       {
+           // we expect to see _some_ errors, so explcitly ignore them
+           var knownPatterns = new List<Regex>
+           {
+               new(@".*Unable to resolve method MongoDB\..*", RegexOptions.Compiled),
+               new(@".*at CallTargetNativeTest\.NoOp\.Noop\dArgumentsIntegration\.OnAsyncMethodEnd.*", RegexOptions.Compiled),
+               new(@".*at CallTargetNativeTest\.NoOp\.Noop\dArgumentsIntegration\.OnMethodBegin.*", RegexOptions.Compiled),
+               new(@".*at CallTargetNativeTest\.NoOp\.Noop\dArgumentsIntegration\.OnMethodEnd.*", RegexOptions.Compiled),
+               new(@".*at CallTargetNativeTest\.NoOp\.Noop\dArgumentsVoidIntegration\.OnMethodBegin.*", RegexOptions.Compiled),
+               new(@".*at CallTargetNativeTest\.NoOp\.Noop\dArgumentsVoidIntegration\.OnMethodEnd.*", RegexOptions.Compiled),
+           };
+
+           var logDirectory = BuildDataDirectory / "logs";
+           if (DirectoryExists(logDirectory))
+           {
+               // Should we care about warnings too?
+               var managedErrors = logDirectory.GlobFiles("**/dotnet-tracer-managed-*")
+                                               .SelectMany(ParseManagedLogFiles)
+                                               .Where(x => x.Level >= LogLevel.Error)
+                                               .Where(IsNewError)
+                                               .ToList();
+
+               var nativeErrors = logDirectory.GlobFiles("**/dotnet-tracer-native-*")
+                                               .SelectMany(ParseNativeLogFiles)
+                                               .Where(x => x.Level >= LogLevel.Error)
+                                               .Where(IsNewError)
+                                               .ToList();
+
+               if (managedErrors.Count == 0 && nativeErrors.Count == 0)
+               {
+                   Logger.Info("No errors found in managed or native logs");
```

Comment:
Presumably this is configured for stdout rather than a file?

Reply:
`Logger.Info()` writes to std out, yes